packages/expo-updates/scripts/createManifest.js

```diff
@@ -44,6 +44,11 @@ const destinationDir = process.argv[4];
   };
 
   assets.forEach(function(asset) {
+    if (!asset.fileHashes) {
+      throw new Error(
+        'It looks like the hashAssetFiles Metro plugin is not configured in your project. You need to add a metro.config.js that configures Metro to use this plugin.'
```

Comment:
maybe we can link to expo.fyi page that has a minimal metro.config.js that includes this?

Reply:
or even readme link is fine actually now that it includes this